### PR TITLE
DOC-807: Update create document collection

### PR DESCRIPTION
**Pull request by diana-macrometa**
Updated the instructions for creating a document collection to reflect the new UI.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
Reviewers, please review.

File: docs/collections/documents/create-document-store.md [hunk 1]
```diff
@@ -17,12 +17,14 @@ Follow these instructions to create a new collection using the GDN console web U
 2. Click **Data > Collections**.
 3. Click **New Collection**.
 4. Click **Document Store**.
-5. Enter information about the collection and then click **Create**.
+5. Enter information about the collection:
 
    - **Collection Name -** Required. A unique name to distinguish the collection. Spaces are not allowed.
-   - **Geo Distribution -** Select whether to store data globally or locally.
+   - **Add date source -** Connect the collection to an external data source. For more information about connections, refer to **Integrations**.
```
Comment (grainier-macrometa) [inline docs/collections/documents/create-document-store.md]:
```suggestion
   - **Add data source -** Connect the collection to an external data source. For more information about connections, refer to **Integrations**.
```

**Review by grainier-macrometa — APPROVED** [review]
@diana-macrometa - Tiny typo, other than that LGTM.

**Review by grainier-macrometa — APPROVED** [review]

**Review by elof — APPROVED** [review]
:shipit:

**Pull request merged**